Source/Core/DolphinQt2/Settings.h

```diff
@@ -51,4 +51,18 @@ class Settings final : public QSettings
   bool GetRenderToMain() const;
   bool GetFullScreen() const;
   QSize GetRenderWindowSize() const;
+
+  // Columns
+  bool& IsBannerVisible() const;
```

Comment:
It is odd to have "Is" functions returning a mutable boolean value.
Can't you have a setter instead of this?